.gitignore

```diff
@@ -2,6 +2,7 @@
 *.egg
 *.so
 *.swp
+*,cover
```

Comment:
I feel this is not needed.

Reply:
I ran:
```sh
tox -e coverage
coverage annotate
```
Which gave me a pile of these mixed in with my changes, but I guess maybe it's more useful to be able to `git clean -f` without `-x`.